diff --git a/node_modules/react-native-actions-sheet/dist/src/types.d.ts b/node_modules/react-native-actions-sheet/dist/src/types.d.ts
index 21bf540..8142e82 100644
--- a/node_modules/react-native-actions-sheet/dist/src/types.d.ts
+++ b/node_modules/react-native-actions-sheet/dist/src/types.d.ts
@@ -1,6 +1,7 @@
 /// <reference types="react" />
 import { ViewStyle } from "react-native";
 export declare type ActionSheetProps = {
+    children: React.ReactNode;
     /**
      * A unique id for the ActionSheet. You must set this if you are using `SheetManager`.
      *
